station/tests/test_db.py

```diff
@@ -22,3 +33,135 @@ def override_get_db():
         yield db
     finally:
         db.close()
+
+def test_crud_docker_train():
+
+    session = TestingSessionLocal()
+
+    list_ids = []
+
+    train_id = str(uuid.uuid4())
+
+    crud_docker_train = CRUDDockerTrain(DockerTrain)
+
+    docker_train_create_obj = DockerTrainCreate(train_id=train_id)
+
+    crud_create_db_train = crud_docker_train.create(session, obj_in=docker_train_create_obj)
+    print("CRUD CreateDockerTrain : {}".format(jsonable_encoder(crud_create_db_train)))
+
+    crud_get_db_train = crud_docker_train.get_by_train_id(session, train_id)
+    print("CRUD DockerTrain : {}".format(jsonable_encoder(crud_get_db_train)))
+
+    docker_train_all = crud_docker_train.get_multi(session)
+
+
+    for docker_train in docker_train_all:
+        print("Docker Train with ID {} exists in the database.".format(docker_train.id))
+        list_ids.append(docker_train.id)
+
+    for n in list_ids:
+        crud_docker_train.remove(session, id=n)
+        print("Docker Train with ID {} got deleted.".format(n))
+
+    session.close()
```

Comment:
maybe missleading but this file is not for tests in regards to the db but for providing a test database (sqlite). Tests for db functionality should live in their own file i.e. test_notifications.py and there you test the crud or api endpoints,